openbas-front/src/admin/components/scenarios/scenario/ScenarioContext.ts

```diff
@@ -11,8 +12,9 @@ const injectContextForScenario = (scenario: ScenarioStore) => {
     onAddInject(inject: Inject): Promise<{ result: string }> {
       return dispatch(addInjectForScenario(scenario.scenario_id, inject));
     },
-    onUpdateInject(injectId: Inject['inject_id'], inject: Inject): Promise<{ result: string }> {
-      return dispatch(updateInjectForScenario(scenario.scenario_id, injectId, inject));
+    async onUpdateInject(injectId: Inject['inject_id'], inject: Inject): Promise<{ result: string }> {
+      await dispatch(updateInjectForScenario(scenario.scenario_id, injectId, inject));
+      return dispatch(fetchScenarioTeams(scenario.scenario_id));
```

Comment:
Why we need to refetch scenario teams here ?
Is not really performant

Reply:
Sure! I agree with you, but we need to get the teams updated here. We have to identify technical teams, injects without teams, and even teams without injects. For that, I re-fetch the teams. However, I could try passing just the injects and see what we can show.

Reply:
The injects contains the new team ids but it's maybe not straightforward to re map to technical teams, injects without teams, and even teams without injects.
This is what you mean ?

Reply:
Fixed with Romuald modifs